mobile/src/main/java/org/openhab/habdroid/ui/AbstractBaseActivity.kt

```diff
@@ -120,16 +100,64 @@ abstract class AbstractBaseActivity : AppCompatActivity(), CoroutineScope {
     }
 
     fun checkFullscreen(isEnabled: Boolean = isFullscreenEnabled) {
-        var uiOptions = window.decorView.systemUiVisibility
-        val flags = (View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
-            or View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY
-            or View.SYSTEM_UI_FLAG_FULLSCREEN)
-        uiOptions = if (isEnabled && !forceNonFullscreen) {
-            uiOptions or flags
+        @Suppress("DEPRECATION")
+        fun checkFullscreenPreR() {
+            var uiOptions = window.decorView.systemUiVisibility
+            val flags = (View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
```

Comment:
Stickler comments actually seem valid here (or at least can be followed without making the code look ugly)